src/App.tsx

```diff
@@ -1,24 +1,88 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+} from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { getTodos } from './api/todos';
+import { Footer } from './components/Footer';
+import { prepareTodos } from './helpers';
+import { Header } from './components/Header';
+import { ErrorType } from './types/ErrorType';
+import { ErrorNotification } from './components/ErrorNotification';
+import { TodoList } from './components/TodoList';
+import { AppContext } from './AppContext';
+import { USER_ID } from './userId/userId';
 
 export const App: React.FC = () => {
+  const {
+    todos,
+    setTodos,
+    status,
+    setStatus,
+    error,
+    setError,
+  } = useContext(AppContext);
```

Comment:
You can create a custom hook to avoid passing context to useContext function every time
```typescript
const useAppContext = () => useContext(AppContext);
```